### PR TITLE
Add description for specification's intended audience

### PR DESCRIPTION
**Pull request by jacoscaz**
This PR refactors #29 on top of the BikeShed transition brought in by #45 . Content-wise it should be identical to the former.

### CODE REVIEW

File: spec/identity/index.bs [hunk 1]
```diff
@@ -130,6 +130,14 @@ manner. Such a web of trust can then be used by a [=Service=] to make
...
 properties of an agent, such as that they are known by some relevant people,
 employed at a given company, a member of a family or some other group, etc.
+
+This specification is for:
+
+- Anyone that wants to understand the architectural principles and notions underlying WebID.
+- URI owners who want to allocate identifiers to denote agents.
+- Resource server developers who want to enable clients to consume WebIDs and their associated descriptions.
+- Application developers who want to implement a client to consume WebIDs.
+- Specification authors who want to integrate or extend WebID.
```
Comment (TallTed) [inline spec/identity/index.bs:140]:
```suggestion
- Anyone who wants to understand the architectural principles and notions underlying WebID
- Content publishers who want to allocate identifiers to unambiguously denote agents
- Server application developers who want to provide client applications with WebIDs and associated agent descriptions
- Client application developers who want to use a WebID to unambiguously identify their applications as agents
- Specification authors who want to extend WebID via additional specifications, e.g., identity authentication protocols
```

**Comment by TallTed** [comment]
> @TallTed and @kidehen, could you please re-apply the changes you requested in https://github.com/w3c/WebID/pull/29#discussion_r1447926962 on top of this one?

[Applied](https://github.com/w3c/WebID/pull/56#discussion_r1473242136).

**Comment by jacoscaz** [comment]
_/chair hat on_

This PR originated from #28 , which has been open for almost a month and already reviewed by a few of us. Furthermore, it does not introduce a significant change. Deadline set to 2024-02-05.

**Comment by TallTed** [comment]
> This PR originated from https://github.com/w3c/WebID/issues/28

Nope, #29

**Comment by jacoscaz** [comment]
Thanks @TallTed !

**Comment by melvincarvalho** [comment]
Given WebID is supposed to be a micro spec, I dont think it's a good idea to add more to it *at this stage*.

Now:

> Specification authors who want to extend WebID via additional specifications, e.g., identity authentication protocols

This is also unclear.  Shouldnt the primary way to extend WebID in 2024 be through extension profiles.  I think we should do that first, before making the micro spec, less micro.

**Comment by jacoscaz** [comment]
_/chair hat off, emphasis mine_

> > Specification authors who want to extend WebID via additional specifications, e.g., *identity authentication protocols*
> 
> This is also unclear. Shouldnt the primary way to extend WebID in 2024 be *through extension profiles*.

@melvincarvalho "additional specifications" should become "extension profiles" for sure, good catch. 

_/chair hat on_

> I dont think it's a good idea to add more to it at this stage

@melvincarvalho could you clarify whether this is a strong objection or whether you could live with this addition, though as a compromise rather than your favorite way forward? 

For context, I do agree that we should focus on implementing the consensus represented by #37 (I'm going to explicitly talk about this in the next _chair's overview_). That said, I see this PR as part of the "backlog" that preceded our recent work on process, a backlog that I'm slowly working through to make room for what comes next.

**Comment by jacoscaz** [comment]
_/chair hat on_

Today is the deadline for review of this PR, which builds upon #29 . @csarven , @kidehen and @TallTed have actively participated in it and no objections have come in - merging. That said, I note that:

- @melvincarvalho might have a strong objection and, if so, I'll make sure to record it in my chair's notes.
- @webr3 would rather we focus on creating new documents rather than iterating on the current ones. However, the majority of this group prefers the latter and, while I _personally_ actually agree with @webr3, I must accomodate the group at large or at least give it my best go before changing strategy.

**Pull request merged**